moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -1326,42 +1364,45 @@ void PlanningSceneMonitor::configureCollisionMatrix(const planning_scene::Planni
 {
   if (!scene || robot_description_.empty())
     return;
-  collision_detection::AllowedCollisionMatrix& acm = scene->getAllowedCollisionMatrixNonConst();
+  // TODO: Uncomment when XmlRpc is refactored
+  // collision_detection::AllowedCollisionMatrix& acm = scene->getAllowedCollisionMatrixNonConst();
 
   // read overriding values from the param server
 
   // first we do default collision operations
-  if (!nh_.hasParam(robot_description_ + "_planning/default_collision_operations"))
-    ROS_DEBUG_NAMED(LOGNAME, "No additional default collision operations specified");
+  if (!node_->has_parameter(robot_description_ + "_planning/default_collision_operations"))
+    RCLCPP_DEBUG(LOGGER, "No additional default collision operations specified");
   else
   {
-    ROS_DEBUG_NAMED(LOGNAME, "Reading additional default collision operations");
-
-    XmlRpc::XmlRpcValue coll_ops;
-    nh_.getParam(robot_description_ + "_planning/default_collision_operations", coll_ops);
-
-    if (coll_ops.getType() != XmlRpc::XmlRpcValue::TypeArray)
-    {
-      ROS_WARN_NAMED(LOGNAME, "default_collision_operations is not an array");
-      return;
-    }
-
-    if (coll_ops.size() == 0)
-    {
-      ROS_WARN_NAMED(LOGNAME, "No collision operations in default collision operations");
-      return;
-    }
-
-    for (int i = 0; i < coll_ops.size(); ++i)  // NOLINT(modernize-loop-convert)
-    {
-      if (!coll_ops[i].hasMember("object1") || !coll_ops[i].hasMember("object2") || !coll_ops[i].hasMember("operation"))
-      {
-        ROS_WARN_NAMED(LOGNAME, "All collision operations must have two objects and an operation");
-        continue;
-      }
-      acm.setEntry(std::string(coll_ops[i]["object1"]), std::string(coll_ops[i]["object2"]),
-                   std::string(coll_ops[i]["operation"]) == "disable");
-    }
+    RCLCPP_DEBUG(LOGGER, "Reading additional default collision operations");
```

Comment:
Is this part important anymore, after googling a little bit seems to me this was part of the arm_navigation package .? see [1](http://library.isr.ist.utl.pt/docs/roswiki/arm_navigation(2f)Tutorials(2f)tools(2f)Understanding(20)and(20)adjusting(20)the(20)autogenerated(20)arm_navigation(20)application.html) [2](http://wiki.ros.org/planning_environment/YAML%20configuration)